mainDoc.html

```diff
@@ -0,0 +1,544 @@
+                    style="width: 591px; height: 350px; border:0; transform: scale(1); overflow:hidden;" sandbox="allow-scripts allow-same-origin">
+            </iframe><br>
+            </item>
+
+            <!-- badge on icons -->
+            <div class="bdg-div-holder">
+                <h3> >> Badge On Icons </h3><br><br>
+                <div class="bdg-holder">
+                    <i class="fas fa-heart fa-2x"></i>
+                    <div class="bdg-icon bdg-top primary-bg">3</div>
+                </div>
+
+                <div class="bdg-holder">
+                    <i class="fas fa-envelope fa-2x"></i>
+                    <div class="bdg-icon bdg-top secondary-bg">1</div>
+                </div>
+
+                <div class="bdg-holder">
+                    <i class="fas fa-calendar fa-2x "></i>
+                    <div class="bdg-icon bdg-top red-bg">9</div>
+                </div>
+
+            </div>
+
+            <item class="frame">
+                <iframe src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=lucario&wt=none&l=css&ds=false&dsyoff=20px&dsblur=68px&wc=false&wa=true&pv=0px&ph=0px&ln=true&fl=1&fm=Hack&fs=13px&lh=139%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522bdg-holder%2522%253E%250A%2520%2520%2520%253Ci%2520class%253D%2522fas%2520fa-heart%2520fa-2x%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%253Cdiv%2520class%253D%2522bdg-icon%2520bdg-top%2520primary-bg%2522%253E3%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%250A%250A%253Cdiv%2520class%253D%2522bdg-holder%2522%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522fas%2520fa-envelope%2520fa-2x%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522bdg-icon%2520bdg-top%2520secondary-bg%2522%253E1%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%250A%250A%253Cdiv%2520class%253D%2522bdg-holder%2522%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522fas%2520fa-calendar%2520fa-2x%2520%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522bdg-icon%2520bdg-top%2520red-bg%2522%253E9%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+                    style="width: 490px; height: 296px; border:0; transform: scale(1); overflow:hidden;" sandbox="allow-scripts allow-same-origin">
+                </iframe>
+                <br>
+            </item>
+
+
+
+
+            <!-- badge on Cards -->
+            <div class="row">
+                <h3> >> Badge On Cards </h3>
+                <div>
+                    <item class="primary-bg cd-bdg white-font">Trending</item>
```

Comment:
```suggestion
                    <item class="primary-bg bdg-card white-font">Trending</item>
```
As you are using bdg as a prefix so you can follow the same naming convention.